### PR TITLE
FLA-383 modify api to display card on file

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- FLA-383 - adding flag to know if user has a registered card
- refactor api spec

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Running application locallly

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/jag-efiling-api.yaml [hunk 1]
```diff
@@ -223,14 +223,14 @@ paths:
...
-              $ref: "#/components/schemas/UserDetails"
+              $ref: "#/components/schemas/CreateCsoAccountRequest"
       responses:
         "201":
           description: Account Created
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/UserDetails"
+                $ref: "#/components/schemas/UserFullDetails"
```
Comment (sdevalapurkar-bcgov) [inline src/backend/efiling-api/jag-efiling-api.yaml:233]: question: why was this changed to `userfulldetails`? how come `userdetails` wasnt a good name for the model?

Comment (alexjoybc) [reply src/backend/efiling-api/jag-efiling-api.yaml:233]: userdetails is used as well, the full details includes first name last name ...
